README.md

```diff
@@ -15,16 +15,16 @@
 ![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/metalbear-co/mirrord)
 [![Twitter Follow](https://img.shields.io/twitter/follow/metalbearco?style=social)](https://twitter.com/metalbearco)
 
-mirrord lets you easily mirror traffic from your Kubernetes cluster to your development environment. It comes as both a [Visual Studio Code](https://code.visualstudio.com/) extension and a CLI tool.
+mirrord lets developers run local processes in the context of their cloud environment. It’s meant to provide the benefits of running your service on a cloud environment (e.g. staging) without actually going through the hassle of deploying it there, and without disrupting the environment by deploying untested code. It comes as a [Visual Studio Code](https://code.visualstudio.com/) extension, an [IntelliJ](https://www.jetbrains.com/products/) plugin and a CLI tool. You can read more about it [here](https://mirrord.dev/docs/overview/introduction/).
```

Comment:
maybe instead of linking to the products we'll link to the extensions? it's probably the expectation of the clicker?

Reply:
I'll just remove the links, they're confusing. The products are linked in like the next line.